ui/kotlinx-coroutines-android/test/R8ServiceLoaderOptimizationTest.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2016-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.android
+
+import org.jf.baksmali.Adaptors.ClassDefinition
+import org.jf.baksmali.BaksmaliOptions
+import org.jf.dexlib2.DexFileFactory
+import org.jf.dexlib2.iface.ClassDef
+import org.jf.dexlib2.iface.DexFile
+import org.jf.util.IndentingWriter
+import org.junit.Test
+import java.io.File
+import java.io.StringWriter
+import java.util.stream.Collectors
+import kotlin.test.assertEquals
+
+class R8ServiceLoaderOptimizationTest {
```

Comment:
Can you, please reformat it to Kotlin Libraries code style:
It is basically standard Kotlin style (4 spaces, etc), but we also use `*` imports always.